### PR TITLE
Adding very basic header.

### PR DESCRIPTION
**Pull request by H-L**
I need to add style, but it's working. I'm currently looking to add **Bootstrap 4** or **Material UI** for v0.1, just to be style efficient quickly.

(I know it's a bit poor but I've worked on other stuffs tonight, sorry 🙏)
I'll catch up in the next few days.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -15,6 +15,10 @@
     <% flash.each do |key, value| %>
       <div class="alert alert-<%= key %>"><%= value %></div>
     <% end %>
+
```
Comment (armandfardeau) [inline app/views/layouts/application.html.erb]: Please remove empty lines

Comment (H-L) [reply app/views/layouts/application.html.erb]: This is less readable, why removing it ?

**Comment by armandfardeau** [comment]
Thanks @H-L It can be merged !

**Comment by H-L** [comment]
Fixes #32

**Pull request merged**